packages/indy-vdr/tests/indy-vdr-pool.e2e.test.ts

```diff
@@ -0,0 +1,243 @@
+                  n: '95671911213029889766246243339609567053285242961853979532076192834533577534909796042025401129640348836502648821408485216223269830089771714177855160978214805993386076928594836829216646288195127289421136294309746871614765411402917891972999085287429566166932354413679994469616357622976775651506242447852304853465380257226445481515631782793575184420720296120464167257703633829902427169144462981949944348928086406211627174233811365419264314148304536534528344413738913277713548403058098093453580992173145127632199215550027527631259565822872315784889212327945030315062879193999012349220118290071491899498795367403447663354833',
+                  s: '1573939820553851804028472930351082111827449763317396231059458630252708273163050576299697385049087601314071156646675105028237105229428440185022593174121924731226634356276616495327358864865629675802738680754755949997611920669823449540027707876555408118172529688443208301403297680159171306000341239398135896274940688268460793682007115152428685521865921925309154307574955324973580144009271977076586453011938089159885164705002797196738438392179082905738155386545935208094240038135576042886730802817809757582039362798495805441520744154270346780731494125065136433163757697326955962282840631850597919384092584727207908978907',
+                },
+              },
+              signature_type: 'CL',
+              ref: schemaResponse.result.txnMetadata.seqNo,
+              tag: 'TAG',
+            },
+          },
+        },
+      })
+    })
+
+    test('fails writing a schema with existing verson number using the pool', async () => {
+      const pool = indyVdrPoolService.getPoolForNamespace('pool:localtest')
+
+      const dynamicVersion = `1.${Math.random() * 100}` // TODO Remove this before pushing
+
+      const schemaRequest = new SchemaRequest({
+        submitterDid: 'TL1EaPFCZ8Si5aUrqScBDt',
+        schema: {
+          id: 'test-schema-id',
+          name: 'test-schema',
+          ver: '1.0',
+          version: dynamicVersion,
+          attrNames: ['first_name', 'last_name', 'age'],
+        },
+      })
+
+      const schemaRequest2 = new SchemaRequest({
+        submitterDid: 'TL1EaPFCZ8Si5aUrqScBDt',
+        schema: {
+          id: 'test-schema-id',
+          name: 'test-schema',
+          ver: '1.0',
+          version: dynamicVersion,
+          attrNames: ['first_name', 'last_name', 'age'],
+        },
+      })
+
+      const response = await pool.submitWriteRequest(agentContext, schemaRequest, signerKey)
+      expect(response).toBeDefined()
+    })
```

Comment:
This test doesn't test what it says it does. I think we can just remove it for now